jepler_udecimal/utrig.py

```diff
@@ -218,10 +224,15 @@ def acos(x, context=None):
 
     with localcontext(context) as ctx:
         ctx.prec += 2
-        r = atan((1 - x * x).sqrt() / x)
+        if x == 1:
+            r = Decimal(0). # 0 radians
+        elif x == -1:
+            r = Decimal(1).atan() * 4  # pi radians
+        else:
+            r = atan((1 - x * x).sqrt() / x)
         if r < 0:
             r += 4 * atan(1)
-        return r
+        return r / 1
```

Comment:
It's a preexisting bug but shouldn't the return r/1 be outside the local context with statement?

Reply:
Yes. Fixed in the latest. Thank you for your keen eye.